bench/workgen/workgen.i

```diff
@@ -174,6 +174,17 @@ WorkgenFrozenClass(WorkloadOptions)
         if config == None:
             config = self.default_config
         self.initialize()
+        bucketConfigIndex = config.find("bucket=")
```

Comment:
A comment would be useful here. I'm guessing we are doing this to ensure we create a directory with the specified bucket name? What is the expected format for `bucket`?

Reply:
Fixed in 33f9f0bce4531ab956baf6445468334309a0d342